dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -290,6 +290,11 @@ namespace dwa_local_planner {
       ROS_DEBUG("flag reset due to set plan at range %f", divergenceDistance);
       oscillation_costs_.resetOscillationFlags();
     }
+    actual_global_plan_.resize(orig_global_plan.size());
```

Comment:
If global plan is a vector, you should be able to just use the copy assignment operator, i.e. actual_global_plan_ = orig_global_plan

Reply:
Will change to make it cleaner.